sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsModelFactory.cs

```diff
@@ -258,20 +258,6 @@ public static DetectLanguageResultCollection DetectLanguageResultCollection(IEnu
         #endregion Detect Language
 
         #region Recognize Entities
-        /// <summary>
```

Comment:
let's not remove this. This guarantees we are not going to break customers that are using the previous version of the service (v3.0).
You don't have to add anything to that method

Reply:
sorry, i checked the reference for both methods, and they were referenced by no one!
i assume it's only us who're using these methods (or ModeFactory) in general

Reply:
yes! no worries :)

Reply:
ok reverted!